projects/packages/forms/src/blocks/field-consent/edit.js

```diff
@@ -130,6 +131,13 @@ export default function ConsentFieldEdit( props ) {
 		__unstableMarkNextChangeAsNotPersistent,
 	] );
 
+	// Ensure the className is set to 'is-style-list' if it is empty or not set.
+	useEffect( () => {
+		if ( className === '' || ! className ) {
+			setAttributes( { className: 'is-style-list' } );
+		}
+	}, [ className, setAttributes ] ); // This effect is a placeholder for any future side effects.
+
```

Comment:
@enejb - I think we removed the list vs browser styles, so just wanted to confirm we still want the useEffects?

Reply:
Yes this effect add the `is-style-list` class which updates the checkbox on save.

Reply:
Does this invalidate the document on open and require saving before closing editor?